template/build/dev-server.js

```diff
@@ -12,6 +12,7 @@ var webpackConfig = {{#if_or unit e2e}}process.env.NODE_ENV === 'testing'
 
 // default port where dev server listens for incoming traffic
 var port = process.env.PORT || config.dev.port
+var autoOpenBrowser = config.dev.autoOpenBrowser || true
```

Comment:
Sorry, my mistake. I mean set default to `true`, but seems set to `false` is more concise. I will pull a new request and close this because the repository is deleted.